provisioning/resources/init/caddy_init

```diff
@@ -31,6 +31,7 @@ case "$1" in
         echo "Already started"
     else
         echo "Starting $name"
+        cd $dir
```

Comment:
Rather than changing directory why not make the `cmd` incorporate the directory?

```
cmd="${dir}/caddy -conf /home/ubuntu/snowplow/configs/Caddyfile"
```